client/rest-high-level/src/test/java/org/opensearch/client/RankEvalIT.java

```diff
@@ -151,7 +151,13 @@ public void testMetrics() throws IOException {
             DiscountedCumulativeGain::new,
             () -> new ExpectedReciprocalRank(1)
         );
-        double expectedScores[] = new double[] { 0.4285714285714286, 1.0, 0.75, 1.6408962261063627, 0.4407738095238095 };
+        double expectedScores[];
```

Comment:
@Sunidhi-Gaonkar1 I see that difference are very negligible, we could improve assertion to take these precision differences into account

Reply:
Yes, it should probably just be increasing delta (presently Double.MIN_VALUE to some expected decimal value) in assertion:

```assertEquals(expectedScores[i], response.getMetricScore(), Double.MIN_VALUE);```